src/pages/Home/CustomizeBobaModalBody.tsx

```diff
@@ -21,9 +44,39 @@ const BobaModalform = ({ milk, setMilk }: { milk: string; setMilk: (milk: string
         <Col xs={12} md={8}>
           Milk(Required)
           <Form.Group>
-            <CustomizeRadioButton milkList={milkList} milk={milk} setMilk={setMilk} />
+            {Object.keys(milkList).map((key) => {
+              const milkOption = milkList[key];
+              return (
+                <CustomizeRadioButton
+                  type={'radio'}
+                  id={key}
+                  key={key}
+                  label={milkOption.name}
+                  handlechange={radioBoxHandler}
+                  check={milk === key}
```

Comment:
Smart thinking

Reply:
LMAO i try using the helper function and didnt work so i change it to this LMAO